vpr/src/place/place.cpp

```diff
@@ -3067,6 +3071,10 @@ void stop_placement_and_check_breakopints(t_pl_blocks_to_be_moved& blocks_affect
 
```

Comment:
Consider if this routine should be moved to another file, in /draw or in a placer breakpoints .cpp file (can't recall the exact names).